databricks/koalas/tests/test_dataframe.py

```diff
@@ -1699,14 +1699,15 @@ def test_drop_duplicates(self):
         pdf = pd.DataFrame({'a': [1, 2, 2, 2, 3], 'b': ['a', 'a', 'a', 'c', 'd']})
         kdf = ks.from_pandas(pdf)
 
+        # inplace is False
         self.assert_eq(pdf.drop_duplicates().sort_index(),
                        kdf.drop_duplicates().sort_index())
         self.assert_eq(pdf.drop_duplicates('a').sort_index(),
                        kdf.drop_duplicates('a').sort_index())
         self.assert_eq(pdf.drop_duplicates(['a', 'b']).sort_index(),
                        kdf.drop_duplicates(['a', 'b']).sort_index())
 
-        # multi-index columns
+        # multi-index columns, inplace is False
```

Comment:
Not pretty sure, but IMO, maybe this line is not necessary since there are also another options not only `inplace`

Reply:
It's alright as long as it's easier to read. explicitly classifying the case is okie.

Reply:
@HyukjinKwon thanks for the comment!